### PR TITLE
mrc-5049: Remove data exploration

### PR DESCRIPTION
**Pull request by r-ash**
## Description

We've seen in the logs no users accessing data exploration mode and UNAIDS have stopped telling people about it as they thought it was confusing. 

This PR removes data exploration mode which should hopefully remove a bit of maintenance overhead

## Type of version change

Minor

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

**Review by EmmaLRussell — COMMENTED** [review]
LGTM!

**Review by M-Kusumgar — APPROVED** [review]
LGTM pending emmas comments! wow this really was scattered in a bunch of places

File: src/app/src/test/kotlin/org/imperial/mrc/hint/integration/HintApplicationTests.kt [hunk 1]
```diff
@@ -83,41 +76,6 @@ class HintApplicationTests : SecureIntegrationTests()
                 .isEqualTo("/login?username=test.user%40example.com&error=BadCredentialsException")
     }
 
-    @Test
-    fun `redirects to requested url after login`()
-    {
-        // set requested url
-        val loginEntity = testRestTemplate.getForEntity<String>("/login?redirectTo=explore")
```
[the remark this answers is not in the document]
Comment (r-ash) [reply src/app/src/test/kotlin/org/imperial/mrc/hint/integration/HintApplicationTests.kt:90]: Yeah I noticed this wasn't working on production either. I've added a ticket and linked to this comment so we can fix this up in a 2nd PR https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5063/Fix-links-to-accessibility-and-privacy-from-landing-page

File: src/app/static/src/tests/e2e/login.etest.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {test, expect, Page} from '@playwright/test';
+
+test.describe("data exploration test", () => {
+
+    test('can redirect to exploration page', async ({page}) => {
```
[the remark this answers is not in the document]
Comment (r-ash) [reply src/app/static/src/tests/e2e/login.etest.ts]: Thanks, fixed that

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**